src/lib/parser_txdef.h

```diff
@@ -45,6 +46,17 @@ typedef struct {
     size_t len;
 } quantity_t;
 
+typedef struct {
+    // REVIEW: signature is 64 bytes
+    uint8_t buffer[64];
+    size_t len;
```

Comment:
I dont think we need len here. it should always be 64 and we should check for this when parsing.